src/features/configTable/configTable.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from "react";
+import { Field, Form, FormRenderProps } from "react-final-form";
+import arrayMutators from "final-form-arrays";
+import { FieldArray } from "react-final-form-arrays";
+import {
+  Box,
+  Button,
+  Checkbox,
+  HStack,
+  FormLabel,
+  Input,
+  VStack,
+  Heading,
+  Spacer,
+  IconButton,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+  FormErrorMessage,
+  FormControl,
+  FormHelperText,
+} from "@chakra-ui/react";
+import { FaPlus, FaTrash } from "react-icons/fa";
+
+type ValidationErrorType = {
+  configData: Employee[];
+};
+
+export type Employee = {
+  name: string;
+  surName: string;
+  employed: boolean;
+};
+
+const onSubmit = async (values: { configData: Employee[] }) => {
```

Comment:
please show how to wire it api mutations since the use case is to call into a remote api. 

you can use a custom queryFn to noop in RTKQ.

Add a delay of 2sec to simulate network and show how to handle disable buttons, spinners and all the needed details to make it a real case.

https://redux-toolkit.js.org/rtk-query/usage/customizing-queries#using-a-no-op-queryfn